### ISSUE TITLE
Validate model during AL loop

### ISSUE THREAD

**Issue by igorsieradzki** [gmum/mlls2015#3]
- with external test set
- with unlabeled portion of training data
